pkg/acsengine/defaults-kubelet.go

```diff
@@ -85,6 +85,7 @@ func setKubeletConfig(cs *api.ContainerService) {
 		for _, key := range []string{"--anonymous-auth", "--client-ca-file"} {
 			delete(o.KubernetesConfig.KubeletConfig, key)
 		}
+		o.KubernetesConfig.KubeletConfig["--authorization-mode"] = "AlwaysAllow"
```

Comment:
not RBAC?

Reply:
I believe it's the authorization mode for the requests from API server to kubelet.

Reply:
Yes. That is right. Without AlwaysAllow, kubectl logs will fail.